cmd/celestia-appd/cmd/addrbook.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/p2p/pex"
+)
+
+const (
+	// routabilityStrict is a hard-coded config value for the address book.
+	// See https://github.com/celestiaorg/celestia-core/blob/793ece9bbd732aec3e09018e37dc31f4bfe122d9/config/config.go#L540-L542
+	routabilityStrict = true
+)
+
+func addrbookCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "addrbook peers.txt addrbook.json",
+		Short: "Convert a list of peers into an address book",
+		Long: "Convert a list of peers into an address book.\n" +
+			"The first argument (peers.txt) should contain a new line separated list of peers. The format for a peer is `id@ip:port` or `id@domain:port`.\n" +
+			"The second argument (addrbook.json) should be the output filename. The address book is saved to the output file in JSON format.\n",
```

Comment:
[Suggestion] I tried the command locally, and can say the second argument can include path to the output file as well.  It might be good to mention it in the help description. 
```suggestion
			"The second argument (addrbook.json) should contain the complete file path, including both the directory path and the desired output file name, in the following format: `path/to/filename`. The address book is saved to the output file in JSON format.\n",
```

Reply:
[Suggestion] It would be also beneficial to indicate in the description where would be the default location for the output file if no path is given.

Reply:
It's a requirement argument so there is no default location.